packages/autocomplete-core/src/utils/isSamsung.ts

```diff
@@ -0,0 +1,5 @@
+const regex = /((gt|sm)-|galaxy nexus)|samsung[- ]([-\w]+)/i;
```

Comment:
We don't need to capture what comes after the Samsung identifiers, and this shaves a few bytes.

```suggestion
const regex = /((gt|sm)-|galaxy nexus)|samsung[- ]/i;
```